src/main/java/com/zenjava/javafx/maven/plugin/WebMojo.java

```diff
@@ -163,6 +165,11 @@
      */
     protected String keyStoreType;
 
+    /**
+     * @parameter

```

Comment:
Required description of this parameter. Seems like you already done it PR description. Just need to convert it to JavaDoc.